File: estudo/neri/algoritmo/c/Cheque.c
#include <stdio.h>
int main(){
    float saldo, cheque_entrou, saldo_final;
    saldo = 500;
    printf(" Digite o valor do cheque que entrou.: ");
    scanf("%f",&cheque_entrou);
    saldo_final = saldo - cheque_entrou;
    printf("\nInforme o saldo: %f",saldo);
    printf("\nInforme o saldo final: %f",saldo_final);
}

File: estudo/neri/algoritmo/java/Cheque.java
import javax.swing.JOptionPane;

public class Cheque {
    public static void main(String args[]){
        float saldo, entrada_cheque, saldo_final;
        saldo = 500;
        entrada_cheque = Float.parseFloat(JOptionPane.showInputDialog(null,"Informe o valor do cheque:"));
        saldo_final = saldo - entrada_cheque;
        JOptionPane.showMessageDialog(null,"Informe o saldo final"+saldo_final);
    }
}
